common-module/src/main/resources/application.yml

```
@@ -0,0 +1,17 @@
+spring:
+  h2:
+    console:
+      enabled: true
+  datasource:
+    url: jdbc:mysql://localhost:3306/test?useSSL=false&characterEncoding=UTF-8&serverTimezone=UTC&createDatabaseIfNotExist=true
+    driver-class-name: com.mysql.cj.jdbc.Driver
+    username: root
+    password:
+  jpa:
+    database-platform: mysql
```

mysql 5.xx 버전을 사용중이라면 아래처럼 작성해보시고, 8.xx 버전이시면 아래에 5를 8로 수정후 한번 해보세요.
그래도 ddl-auto: create가 안되신다면 개발에는 h2 데이터베이스(디폴트값 = create)가 효율적이니깐 한번 해보시면 될것 같네요.
```suggestion
    database-platform: org.hibernate.dialect.MySQL5Dialect
```

---

아니면 resource 디렉터리에 init.sql 하나 생성해서 직접 쿼리 작성해도 될거에요.